docs/usage/configuration-options.md

```diff
@@ -1862,21 +1862,21 @@ For example, if you have an `examples` directory and you want all updates to tho
 {
   "packageRules": [
     {
-      "matchPaths": ["examples/**"],
+      "matchFileNames": ["examples/**"],
       "extends": [":semanticCommitTypeAll(chore)"]
     }
   ]
 }
 ```
 
-If you wish to limit Renovate to apply configuration rules to certain files in the root repository directory, you have to use `matchPaths` with a `minimatch` pattern or use [`matchFiles`](#matchfiles) with an exact match.
+If you wish to limit Renovate to apply configuration rules to certain files in the root repository directory, you have to use `matchFileNames` with a `minimatch` pattern (which can include an exact file name match).
```

Comment:
Line 1858 mentions:

> Path rules are convenient to use if you wish to apply configuration rules to certain package files using patterns.

I think you should search for `path` as well in the `*.md` files, you'll probably find more places where `path` should be replaced with `matchFileNames` or a similar word.